ruby_review_week_four.rb

```diff
@@ -1,22 +1,69 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+#variables are objects that can be stored and referred to again and again. 
 
 # 2. What Ruby method can you use to make a loop from an array?
 
-# 3. In Ruby, what are booleans?
+#.each , .map
+
+# 3. In Ruby, what are booleans? 
+#true or false 
 
 # 4. How do classes relate to objects?
 
+#classes are the blueprint for objects
+
 # 5. What is an instance variable?
 
+#they are class attributes
+
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
 
+# class Person
+
+# def initialize(haircolour, eyecolour)
+#   @haircolour = haircolour
+#   @eyecolour = eyecolour
+
+# end 
+
+#   def description
+#     return "Haircolour: #{@haircolour} and Eyecolour: #{@eyecolour}"
```

Comment:
make sure to follow instructions precisely, the instance method didn't need to be coded in full